step.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"time"
+)
+
+func RunEnvmanAdd(key, value string) error {
+	args := []string{"add", "-k", key, "-v", value}
+	return RunCommand("envman", args...)
+}
+func RunCommand(name string, args ...string) error {
+	cmd := exec.Command(name, args...)
+	cmd.Stdin = os.Stdin
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	return cmd.Run()
+}
+
+func main() {
+	// unix timestamp
+	timestamp := time.Now().Unix()
```

Comment:
Call `time.Now()` only once, don't call it a send time at `timeString := fmt.Sprintf("%v", time.Now().Format(time.RFC3339Nano))`!